test_util/test_util.go

```diff
@@ -0,0 +1,33 @@
+package test_util
+
+import (
+	"fmt"
+	"path/filepath"
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+func Assert(tb testing.TB, condition bool, msg string, v ...interface{}) {
+	if !condition {
+		_, file, line, _ := runtime.Caller(1)
+		fmt.Printf("\033[31m%s:%d: "+msg+"\033[39m\n\n", append([]interface{}{filepath.Base(file), line}, v...)...)
```

Comment:
This doesn't matter at all, but i'd use `[0m` instead of `[39m` (or just `[m`, which I believe is equivalent to `[0m`

`[0m` resets everything. `[39m` only resets the foreground. You're only changing the foreground ... so [39m suffices. But still ... good hygiene. 

http://pueblo.sourceforge.net/doc/manual/ansi_color_codes.html

Reply:
thanks. Removed this in favor of t.Logf